src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -508,6 +535,8 @@ private CommandResult executeCommand(String commandText) throws CommandException
             logger.info("Invalid command: " + commandText);
             resultDisplay.setFeedbackToUser(e.getMessage());
             throw e;
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
```

Comment:
ooh wait this part do we need to prompt users that file does not exist (like through resultDisplay maybe)

Reply:
updated all similar areas